draft-ietf-quic-transport.md

```diff
@@ -2225,14 +2226,11 @@ connection state.  This results in new packets on the connection being handled
 generically.  For instance, an endpoint MAY send a stateless reset in response
 to any further incoming packets.
 
-The draining and closing periods do not apply when a stateless reset
-({{stateless-reset}}) is sent.
-
 An endpoint is not expected to handle key updates when it is closing or
 draining.  A key update might prevent the endpoint from moving from the closing
 state to draining, but it otherwise has no impact.
 
-While in the closing period, an endpoint could receive packets from a new source
+While in the closing period, a server could receive packets from a new source
 address, indicating a client connection migration ({{migration}}). An endpoint
```

Comment:
We've generally avoided assigning roles, preferring to talk about migration as valid in both directions, so it might be better to say endpoint/peer rather than server/client here.  That doesn't matter though.